2-mandatory/2-currency-code-lookup.js

```diff
@@ -16,9 +16,15 @@ const COUNTRY_CURRENCY_CODES = [
   ["NG", "NGN"],
   ["MX", "MXN"],
 ];
-
+newObject = {};
 function createLookup(countryCurrencyCodes) {
   // write code here
+  for (const arr of countryCurrencyCodes){
+    const newKey = arr[0];
+    const newValue = arr[1];
+    newObject[newKey] = newValue; // why doesn't .notation work?
```

Comment:
Using newObject.newKey would create a new property called newKey and overwrite it each time the for of loop iterates. With the .notation, we aren't using the newKey variable on line 23. 

The bracket notation (newObject[newKey] = newValue) uses the variable value on line 23 so it doesn't overwrite as each property has a different name.

Reply:
oh now i get it.Thanks Maten.